vent/menus/main.py

```diff
@@ -415,38 +415,85 @@ def system_commands(self, action):
             # add notify_cancel_ok popup once implemented
             pass
         elif action == "ntapcreate":
-            form_args = {'color': 'CONTROL',
-                         'name': 'Network Tap Interface Create' + "\t"*6 +
-                                 '^T to toggle main'}
-            self.add_form(CreateNTap, "Network Tap Create", form_args)
+            output = self.api_action.tool_status_output('network_tap')
```

Comment:
a lot of repetition inside the `elif`s, can't some of those be moved out and only exist once?

Reply:
You're right. I'll try and reduce the repetition.